cdap-ui/graphql/resolvers/resolvers-common.js

```diff
@@ -20,6 +20,19 @@ function getGETRequestOptions() {
   return {
     method: 'GET',
     json: true,
+    headers: {
+      Authorization: '',
+    },
```

Comment:
Should not be setting Authorization or headers when it doesn't exist

Reply:
This is old code. I removed this in the latest commit